test/test_python_type_stubs.py

```diff
@@ -341,7 +376,7 @@ def test__generate_base_namespace_module__with_nested_types(self):
 
             class NestedTypes(bb.Struct):
                 def __init__(self,
-                             list_of_nullables: Optional[List[Optional[int]]] = ...,
+                             list_of_nullables: List[Optional[int]] = ...,
```

Comment:
Looking at the corresponding code
`            StructField(
                name='NullableList',
                data_type=Nullable(
                    List(UInt64())
                ),
                doc=None,
                ast_node=None,
            ),`

shouldn't this be `List[int]` instead?

Reply:
Actually the definition of this arg `list_of_nullables` is
```
StructField(
                name='ListOfNullables',
                data_type=List(
                    Nullable(UInt64())
                ),
                doc=None,
                ast_node=None,
            )
```
so I think it should be `List[Optional[int]]` here